.github/workflows/lint.yaml

```diff
@@ -80,11 +80,19 @@ jobs:
         id: "goreleaser"
         with:
           distribution: "goreleaser-pro"
-          version: "2.3.2"
+          # NOTE: keep in sync with goreleaser version in other job.
+          # github actions don't allow yaml anchors.
+          version: "v2.12.5"
           args: "release -f .goreleaser.docker.yml --clean --split --snapshot"
         env:
           GORELEASER_KEY: "${{ secrets.GORELEASER_KEY }}"
       - name: "Obtain container image to scan"
-        run: 'echo "IMAGE_VERSION=$(jq .version dist/linux_amd64_v1/metadata.json --raw-output)" >> $GITHUB_ENV'
+        run: |
```

Comment:
FYI
- notice the change from `dist/linux_amd64_v1` to `dist/linux_amd64`, introduced by the new goreleaser version
- made it so that if this step fails, the next one doesn't run